notebooks/lightning_segmentation.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>Maybe use some heavier data augmentation so show high performance for 3D data :) (see comment for training cell)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PhoenixDL/rising/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
I only included few data augmentation, since from my experience users want to get the general concept and not wait forever to finish their training for examples.